Automattic/wp-calypso
# Mark as seen on open post notification

#### Changes proposed in this Pull Request

Mark post as seen when new post notification is open, it should also work for automattcher mentions in posts for a12s.

#### Testing instructions
- `cd apps/notifications && yarn build`
- `yarn start` in Calypso root
- Open notification panel and click on an unread new post notification
- Verify in that in the Network console there is a request for `/seen-posts/seen/blog/new`
- Refresh the page, if possible check if that post is marked as seen in the Reader
